package.json

```
@@ -133,6 +133,7 @@
     "@carbon/icons-react": "10.17.0",
     "@carbon/layout": "10.12.0",
     "@carbon/motion": "10.6.0",
+    "@carbon/pictograms-react": "^11.0.0",
```

Let's pin this like the other Carbon dependencies. 
```suggestion
    "@carbon/pictograms-react": "11.0.0",
```
will need a `yarn install` after to regen the lock file.